### ISSUE TITLE
Setting AI sampling value on config-tool-v2 will instead change offset after reconnect

### ISSUE THREAD

**Issue by fakhrul-omar** [ADFTech/XTEC_UC_MSTP_Public#3]
Due to miss in copy/pasting code without changing variable name. The bug is already identified, just need to test on actual device.

**Comment by fakhrul-omar** [comment]
@kslee911 It should be fine now. Though, from what I'd inspected, MS/TP device with firmware v2.15 and below doesn't have sampling parameter, which will result to ERROR_CODE_UNKNOWN_PROPERTY. Should I disable or hide sampling text box for those versions?

**Comment by kslee911** [comment]
Hide the sampling box would be better. Thanks. Let me know when there's a release.

**Comment by kslee911** [comment]
@fakhrul-omar 
"Due to miss in copy/pasting code without changing variable name" 
Regarding this, can check on other properties as well? It seems like UI always shows unlink, and the UI Type also keep showing -10v

**Comment by fakhrul-omar** [comment]
Nope, there're no problem in code for those properties. Perhaps those happens only when the program started to connect to device (and it doesn't get reply from device for proper value yet)? But for UI type -10V (negative 10V), I cannot find that entry in dropdown list. Is it a typo?

**Comment by kslee911** [comment]
@fakhrul-omar 
There should be no -10V option. This is not required.

**Comment by fakhrul-omar** [comment]
OK then, and so, you can download the latest release at this site:
https://github.com/ADFTech/XTEC_UC_MSTP_Public/releases
or at Dropbox:
(top directory)\Config Tool DL\Config Tool v2

Though this version doesn't have sampling box hiding for previous releases yet. Please wait for a while for that.
